native/cocos/network/HttpClient-apple.mm

```diff
@@ -149,7 +149,15 @@ static int processTask(HttpClient *client, HttpRequest *request, NSString *reque
             unsigned long i = header.find(':', 0);
             unsigned long length = header.size();
             ccstd::string field = header.substr(0, i);
-            ccstd::string value = header.substr(i + 1, length - i);
+            ccstd::string value = header.substr(i + 1);
+            // trim all \n
+            if (!value.empty() && value[value.size() - 1] == '\n') {
```

Comment:
The comment is not correct. It just trim `\n` at the end of string, not all `\n`.